src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -485,20 +485,16 @@ class TcgStatics {
       opp.bench.remove(pcs)
     }
   }
-  static PokemonCardSet benchPCS (Card card, ActivationReason reason=ActivationReason.OTHER, TargetPlayer targetPlayer=TargetPlayer.SELF){
+  static PokemonCardSet benchPCS (Card card, ActivationReason reason=OTHER, TargetPlayer targetPlayer=TargetPlayer.SELF){
     if(card.getCardTypes().is(BREAK)){
```

Comment:
Sorry to bother you again but In my opinion this is another condition check to be moved to the main class.

Reply:
Moved this check over to the PutOnBench effect